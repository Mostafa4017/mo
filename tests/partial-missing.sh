#!/usr/bin/env bash

cd "${0%/*}"
../mo -u partial-missing.template 2>&1
returned=$?

if [[ $returned -ne 1 ]]; then
    echo "Did not return 1. Instead, returned $returned."
fi
