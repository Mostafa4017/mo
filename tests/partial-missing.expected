File does not exist: /home/fidian/repo/mo/tests/partial-missing.partial
